### PR TITLE
[dhcp_server] Write unknown interface name when mac not in fdb to show all valid leases

### PR DESCRIPTION
**Pull request by w1nda**
#### Why I did it
When a client got an assigned IP from dhcp server, there is a record in dhcp server lease file, however we can't show it by cli.

It's because when parsing kea-lease.csv file, we try to find the vlan interface name where mac binding to, if there is no such binding in FDB_TABLE, we will miss some valid leases to write to STATE_DB.

##### Work item tracking
- Microsoft ADO **(number only)**: 27690247

#### How I did it
When mac not in FDB_TABLE, will leave the interface name as Unknown.

#### How to verify it
Add unit test and local build.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_lease.py [hunk 1]
```diff
@@ -120,10 +121,8 @@ def _read(self):
             valid_lifetime = splits[3]
             lease_end = splits[4]
 
-            if mac_address not in fdb_info:
-                syslog.syslog(syslog.LOG_WARNING, "Cannot not find {} in fdb table".format(mac_address))
-                continue
-            new_key = "{}|{}".format(fdb_info[mac_address], mac_address)
+            vlan_name = fdb_info[mac_address] if mac_address in fdb_info else DHCP_UNKNOWN_INTERFACE
```
Comment (yaqiangz) [inline src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_lease.py:124]: Better to use dhcp_interface as var name

Comment (w1nda) [reply src/sonic-dhcp-utilities/dhcp_utilities/dhcpservd/dhcp_lease.py]: Yes, updated.

**Review by yaqiangz — APPROVED** [review]

**Review by matiAlfaro — APPROVED** [review]

**Comment by w1nda** [comment]
Abort this PR due to there is a design change.

We plan to change the way we parse lease, in new implementation, we didn't have this issue.

**Pull request closed without merging**